src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,13 @@
-export const Tabs = () => {};
+export const Tabs = ({
+  tab: { id = 'tab-1', title, content }, setValue, value,
```

Comment:
The default value 'tab-1' for id might not always be accurate or valid. Consider removing it or providing a more meaningful default value.